### PR TITLE
add section about docker configuration tweaks

### PR DESCRIPTION
**Pull request by tomekl007**

### CODE REVIEW

**Comment by jorgebay** [comment]
Maybe we can rephrase it to remove the condition in the sentence, something like:

> Consider that the default computing resources dedicated to Docker might be insufficient. [We recommend a minimum of 8Gb of System Memory for development environments](https://docs.datastax.com/en/dse-planning/doc/planning/planningHardware.html).

**Comment by tomekl007** [comment]
> Maybe we can rephrase it to remove the condition in the sentence, something like:
> 
> > Consider that the default computing resources dedicated to Docker might be insufficient. [We recommend a minimum of 8Gb of System Memory for development environments](https://docs.datastax.com/en/dse-planning/doc/planning/planningHardware.html).

Sounds better but on the other hand, I think that we should leave that problem statement
"If your DSE instance is stopping after starting studio or opsCenter"
because that problem manifests in this particular situation, and users may even CRL+F for those keywords in the README. What do you think?

File: README.md [hunk 1]
```diff
@@ -315,6 +316,13 @@ Head over to [DataStax Academy](https://academy.datastax.com/quick-downloads?utm
 * Step-by-step tutorials and examples
 * How to build applications using Apache Cassandra™/ DataStax
 
+# Docker Configuration Tips and Tricks
+
+* If your DSE instance is stopping after starting studio or opsCenter,
```
Comment (pedjak) [inline README.md]: IMHO, it is not clear what is the usage scenario here - what is the platform you are referring to? what OS? what do you refer as `docker memory`, and how to increase it ?

Comment (tomekl007) [reply README.md]: In this example, I had a problem with MAC OS docker (docker-machine) but every OS could have a different way of changing that. `docker memory` is just a RAM assigned to docker on the host. I thought that there could be a generic section about tips when running our docker-images

Comment (roberd13) [reply README.md]: @tomekl007 If I am reading correctly, you suggesting we add a section on Tweaks and Configuring Docker itself?  From the example you provided, it appears you needed to increase the ram for Docker on Mac allocated to the Virtual Machine the containers run inside, this would also apply to Docker on Windows but not Linux.  

I get you point because I have seen other vendors reference increasing the ram allocated to docker on mac and windows.  Im not sure how deep of a rabbit hole we would want to go down with docker specific settings.

Comment (tomekl007) [reply README.md]: Yes, I added that section: Docker Configuration Tips and Tricks
I think that it may reduce a lot of time for future users because I spend some time figuring out that this is a memory problem. There was no explicit information from docker, it just killed one container

Comment (pedjak) [reply README.md]: I agree with @roberd13 here - @tomekl007 you should expand this paragraph providing the details about the environment where the issue occurred. `docker-machine` is just one way of installing Docker on OSX - [current Docker for OSX](https://docs.docker.com/docker-for-mac/install/) uses HyperKit and a slightly different approach.

Comment (tomekl007) [reply README.md]: Ok :) can you suggest a change? 
There is an info that `Default on OSX is 2GB - that is not enough. `

Comment (pedjak) [reply README.md]: well, just describe as precise as possible your case - I would avoid actually the very general title "Docker Configuration Tips and Tricks" - Instead perhaps just a title stating "OSX Caveats" ?

Comment (tomekl007) [reply README.md]: title changed :)

**Comment by roberd13** [comment]
@tomekl007 Since this does not affect docker on linux.  How about we add under the following under the  prerequisite section? 
When using  Docker for Mac or Docker for Windows, the default resources allocated to the linux VM running docker are 2GB RAM and 2 CPU's. Make sure to adjust these resources to meet the resource requirements for the containers you will be running. More information can be found here on adjusting the resources allocated to docker.

[Docker for mac](https://docs.docker.com/docker-for-mac/#advanced) 
[Docker for windows](https://docs.docker.com/docker-for-windows/#advanced)

**Comment by tomekl007** [comment]
@roberd13 good suggestion -  done
